netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
@erikh - introduced String(); thanks for the hint
@mapuri - simplified some code - PTAL; however given that we'd eventually need a version REST endpoint, I am not inclined to taking that code out.

Let's discuss API versioning separately...